README.org

```diff
@@ -453,6 +456,79 @@ For a more general index related to all-things Emacs, use [[https://github.com/e
 
     Also handles macros, functions defined through macros and let-bound variables.
 
+    If you want to jump to symbols in files which aren't loaded in your Emacs you can fallback to the more general [[https://github.com/jacktasia/dumb-jump][dumb-jump]] package.
+
+
+*** Search and Replace
+
+    In Emacs regular expressions can make use of syntax information provided by the major-mode. This allows for complex symbol searches with =isearch= and =occur=.
+    To search and jump to toplevel definitions of a buffer you can use the built-in =imenu=. The [[https://github.com/vspinu/imenu-anywhere][imenu-anywhere]] package allows to extend the scope to buffers of the same project or mode.
+    It's also worth mentioning the built-in =outline-mode= here which can be used to structure and navigate files by sections. 
```

Comment:
Mhm, wouldn't recommend using literate programming for package (i.e. outside of dot emacs). This can be misleading to newbies.

Reply:
> I don't fully understand the first sentence.

I tried to explain that you can do semantic searches using elisp regexp. For example `M-s .` searches for symbols etc. I will try to expand a bit on that.

> Mhm, regarding outline-mode I wouldn't recommend using literate programming for package (i.e. outside of dot emacs). This can be misleading to newbies

Okay, I removed it.